index.js

```diff
@@ -1,445 +0,0 @@
-      'outside': outside,
-      'kitchen': kitchen,
-      'livingRoom': livingRoom,
-      'gazebo': gazebo,
-      'mansion': mansion,
-      'ballRoom': ballRoom,
-      'bedRoom': bedRoom,
-      'bathRoom': bathRoom,
-    };
-
-
-    /*---------------------------------State Machine------------------------------------------------- */
-    //actual state machine
-    let enterRooms = {
-      'outside': {
-        canChangeTo: ['kitchen']
-      },
-      'kitchen': {
-        canChangeTo: ['outside', 'livingRoom', 'gazebo']
-      },
-      'livingRoom': {
-        canChangeTo: ['kitchen']
-      },
-      'gazebo': {
-        canChangeTo: ['mansion', 'kitchen', 'ballRoom']
-      },
-      'mansion': {
-        canChangeTo: ['gazebo']
-      },
-      'ballRoom': {
-        canChangeTo: ['bedRoom', 'gazebo']
-      },
-      'bedRoom': {
-        canChangeTo: ['bathRoom', 'ballRoom']
-      }
-    }
-
-    //game start location
-    let currentRoom = 'outside'
-
-    //fumction that allows room transitions
-    function changeRooms(newRoom) {
-      let roomTransitions = enterRooms[currentRoom].canChangeTo;
-      if (roomLookUp[newRoom].locked === true) {
-        console.log('The door is locked, move along!');
-      } else if (roomTransitions.includes(newRoom)) {
-        currentRoom = newRoom;
-        let stateForTable = roomLookUp[currentRoom]
-        console.log(stateForTable.desc);
-        zombieHorde()
-        console.log("You better hurry up they're getting closer!" + player.encroacment);
-        player.location = roomLookUp[currentRoom]
-      } else {
-        console.log('You can not make that move from' + currentRoom + 'to' + newRoom)
-      }
-    }
-
-    /*----------------------------------Story--------------------------------------------*/
-    //intro asking if the player would like to play
-    async function intro() {
-      const introMessage = `Welcome to the Zombie Apocalypse! Zombies are all around and closing in fast! Please word your actions in a [action] + [Item/Room] format`;
-      let startPrompt = await ask(introMessage + "\n" + "Do you understand?\n>_");
-      let cleanStart = cleanWords(startPrompt);
-      if (cleanStart === "yes") {
-        start();
-      } else {
-        console.log("Come back when you're ready then!");
-        process.exit();
-      }
-    }
-
-    intro();
-    // game start
-    async function start() {
-      const startMessage = `After some time running through woods, you come upon a gated community that looks overrun and long abandoned. The gate is losely held together by some rusted chains with a note that reads: “We have safety and supplies beyond the town through the tunnel, there is a key in one of the houses that will unlock the mansion at the end of the cul-de-sac and will give you access to the tunnel and make your way to Paradise Cove. Hurry, once the gates are open there is no way to close them and ‘they’ will get in!
-  
-  What would like you to do?`;
-
```

Comment:
In terms of usability, there is no way to tell where you're able to go. I had to look through the code to see that you start 'outside' and that 'outside' is connected to the 'kitchen'.